apps/client/src/shared/api/domain/community/queries.ts

```diff
@@ -153,6 +161,15 @@ export const getCommentReply = async (
   return response;
 };
 
+export const getPopularFeed = async (
+  size?: number,
+): Promise<PopularFeedResponse | null> => {
+  const response = await api
+    .get(END_POINT.COMMUNITY.GET_POPULAR(size))
```

Comment:
END_POINT를 수정하면 이 부분도 이렇게 수정하면 될것 같아요!

```suggestion
    .get(`${END_POINT.COMMUNITY.GET_POPULAR}?size=${size}`)
```

Reply:
👍